RetailCoder.VBE/app.config

```diff
@@ -186,6 +186,24 @@
           </HotkeySetting>
         </value>
       </setting>
+      <setting name="ToDoMarker_ToDo" serializeAs="Xml">
+        <value>
+          <ToDoMarker xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+            xmlns:xsd="http://www.w3.org/2001/XMLSchema" Text="TODO " />
+        </value>
+      </setting>
+      <setting name="ToDoMarker_Note" serializeAs="Xml">
+        <value>
+          <ToDoMarker xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+            xmlns:xsd="http://www.w3.org/2001/XMLSchema" Text="NOTE " />
+        </value>
+      </setting>
+      <setting name="ToDoMarker_Bug" serializeAs="Xml">
+        <value>
+          <ToDoMarker xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+            xmlns:xsd="http://www.w3.org/2001/XMLSchema" Text="BUG " />
+        </value>
+      </setting>
```

Comment:
this is just a non-escaped version of the settings maintained in `RetailCoder.VBE/Properties/Settings.settings`. Why do we have two places where this is version-controlled?

Reply:
@Vogel612 the .settings file is essentially just a designer for the .config file. Shouldn't the markers no longer have a trailing whitespace though?

Reply:
@retailcoder Ah, yes (see #3649). I'll fix this later.